### PR TITLE
External Wallet Component

### PR DESCRIPTION
**Pull request by nalancer08**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Linked issue

https://cybrid.atlassian.net/browse/CYB-1436

### Evidence

https://github.com/Cybrid-app/cybrid-sdk-ios/assets/7268597/1c771ee9-8b57-4019-9cc4-6f857d104961

### QR SCanner

https://github.com/Cybrid-app/cybrid-sdk-ios/assets/7268597/eea3eba6-28c1-44f3-8da8-e35fd6234816

### CODE REVIEW

File: CybridSDK/Components/BankAccounts/BankAccountsViewController.swift [hunk 1]
```diff
@@ -94,7 +94,8 @@ extension BankAccountsViewController {
             switch state {
 
             case .LOADING:
-                self.bankAccountsView_Loading()
+                // self.bankAccountsView_Loading()
```
Comment (brent-cybrid) [inline CybridSDK/Components/BankAccounts/BankAccountsViewController.swift]: Remove

File: CybridSDKTests/Mocks/JSON/listExternalWalletResponseIncomplete.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+            "customer_guid": "stringstringstringstringstringst",
+            "address": "string",
+            "tag": "string",
+            "created_at": "2023-08-23T19:07:20.594Z",
+            "state": "storing",
+            "failure_code": "string"
+        }
+    ]
+}
+
```
Comment (brent-cybrid) [inline CybridSDKTests/Mocks/JSON/listExternalWalletResponseIncomplete.json:18]: An ExternalWallet can also fail to be added. Do you handle that in your views?

File: CybridSDK/Components/BankAccounts/BankAccountsViewController.swift (hunk 1 above)
Comment (nalancer08) [reply CybridSDK/Components/BankAccounts/BankAccountsViewController.swift]: Done!

File: CybridSDKTests/Mocks/JSON/listExternalWalletResponseIncomplete.json (hunk 2 above)
Comment (nalancer08) [reply CybridSDKTests/Mocks/JSON/listExternalWalletResponseIncomplete.json:18]: Done!

File: CybridSDK/Components/ExternalWallets/Extensions/_CreateWallet.swift [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+        warningContainer.addSubview(warningIcon)
+        warningIcon.constraintTop(warningContainer, margin: 10)
+        warningIcon.constraintLeft(warningContainer, margin: 8)
+        warningIcon.constraintWidth(14)
+        warningIcon.constraintHeight(14)
+
+        // -- Warning Label
+        let warningLabel = UILabel()
+        warningLabel.font = UIFont.systemFont(ofSize: 14)
+        // warningLabel.textAlignment = .justified
```
Comment (brent-cybrid) [inline CybridSDK/Components/ExternalWallets/Extensions/_CreateWallet.swift]: Remove

File: CybridSDK/Components/ExternalWallets/ViewController/QRScannerViewController.swift [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+        self.view.addSubview(qrCodeFrameView)
+        qrCodeFrameView.centerVertical(parent: self.view)
+        qrCodeFrameView.centerHorizontal(parent: self.view)
+        qrCodeFrameView.setConstraintsSize(size: CGSize(width: 180, height: 180))
+        self.view.bringSubviewToFront(qrCodeFrameView)
+    }
+
+    func failed() {
+
+        let alert = UIAlertController(title: "Scanning not supported", message: "Your device does not support scanning a code from an item. Please use a device with a camera.", preferredStyle: .alert)
```
Comment (brent-cybrid) [inline CybridSDK/Components/ExternalWallets/ViewController/QRScannerViewController.swift]: Needs to be localized

File: CybridSDK/Components/ExternalWallets/Extensions/_CreateWallet.swift (hunk 3 above)
Comment (nalancer08) [reply CybridSDK/Components/ExternalWallets/Extensions/_CreateWallet.swift]: Done!

File: CybridSDK/Components/ExternalWallets/ViewController/QRScannerViewController.swift (hunk 4 above)
Comment (nalancer08) [reply CybridSDK/Components/ExternalWallets/ViewController/QRScannerViewController.swift]: Done!

**Review by brent-cybrid — APPROVED** [review]

**Pull request merged**